zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/ListViewProcessInstanceFromProcessInstanceHandler.java

```diff
@@ -320,16 +325,24 @@ private void completeOperation(final Record<ProcessInstanceRecordValue> record)
     //    }
   }
 
-  /// TODO - because it depends on processCache
   private String getProcessName(final Long processDefinitionKey, final String bpmnProcessId) {
-    return bpmnProcessId;
+    final var processName =
+        processCache
+            .get(processDefinitionKey)
+            .map(CachedProcessEntity::name)
+            // If the cache does not contain the process definition then the process has been
+            // deleted from the backend. This is a special case which can happen only if there was a
+            // data loss in the backend. In that case, inorder to not block the exporter, we can
+            // return the bpmnProcessId as the process name.
+            .orElse(null);
+    if (processName == null || processName.isEmpty()) {
+      return bpmnProcessId;
+    }
```

Comment:
Was not sure about this because in the original Operate code a blank processName was replaced by `bpmnProcessId` if it is either null or empty. 

With your suggestion, it can return a null value for processName. @sdorokhova What is the expected behavior here?